.github/workflows/check-lambda-js.yml

```diff
@@ -10,15 +10,16 @@ on:
       - main
       - development
       - ia-web-development
+      - ia-development
     paths:
-      - backend/compact-connect/lambdas/cloudfront-csp/**
+      - backend/compact-connect/lambdas/nodejs/**
 
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  CheckLambdaJS:
+  CheckLambdas:
```

Comment:
# Password update validation

Path: `/Account`

The validation messages are all 'green', but the box is red. It's not clear why.

<img width="493" alt="image" src="https://github.com/user-attachments/assets/dff3dc2e-95ad-4f33-ac3e-0c1abc39dd7c" />

Reply:
Do you mind creating a ticket for this one? It might also be good to have the repro steps (unless it basically happens in all cases).